recipe/meta.yaml

```diff
@@ -8,36 +9,34 @@ package:
 source:
   # GitHub release contains tests/
   url: https://github.com/sarugaku/{{ name|lower }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: f400ebd5e60361b3b328241f4aded5a0f7665e46a83c374fea86d874cff0ffd5
+  sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<37]
```

Comment:
I don't see evidence upstream that python 3.6 or older are incompatible. Where did you see that restriction?

Reply:
It's actually not restricted. [This](https://github.com/sarugaku/resolvelib/blob/0.8.1/noxfile.py#L25) is the only requirement and knowing our main focus is 3.7 and up, I did it as a caution. Should I just omit it since I know it builds for 3.8, 3.9, and 3.10(At least on mac)?

Reply:
If there are no restrictions upstream, I don't think we should introduce them without a compelling reason.